ui/app/components/oidc/provider-form.js

```diff
@@ -0,0 +1,72 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency';
+/**
+ * @module OidcProviderForm
+ * OidcProviderForm components are used to create and update OIDC providers
+ *
+ * @example
+ * ```js
+ * <OidcProviderForm @model={{this.model}} />
+ * ```
+ * @callback onCancel
+ * @callback onSave
+ * @param {Object} model - oidc client model
+ * @param {onCancel} onCancel - callback triggered when cancel button is clicked
+ * @param {onSave} onSave - callback triggered on save success
+ */
+
+export default class OidcProviderForm extends Component {
+  @service store;
+  @service flashMessages;
+
+  @tracked modelValidations;
+  @tracked radioCardGroupValue =
+    !this.args.model.allowedClientIds || this.args.model.allowedClientIds.includes('*')
```

Comment:
maybe a comment here about the `includes('*'). Something about how it's not looking for an inputed wildcard value  like 'p*' (which is my first thought), but if empty no clients are allowed and if wildcard all clients are allowed.

Reply:
It's italizing my text and not showing the wildcard. Sorry about the formatting. If you need clarification, ping me.